src/administration/web.md

```diff
@@ -1,35 +1,41 @@
-# Web Interface
+# Management Console
 
-Platform.sh provides a responsive Web Interface which allows you to interact with your projects and manage your environments. Everything you can do with this interface you can also achieve with the  [CLI (Command Line Interface)](/gettingstarted/cli.md) and vice versa.
+Platform.sh provides a responsive management console which allows you to interact with your projects and manage your environments. 
 
-![Web Interface](/images/platform-ui.png)
+![Management Console](/images/Project.png)
+
+Everything you can do with the management console you can also achieve with the  [CLI (Command Line Interface)](/gettingstarted/cli.md) and vice versa.
 
 ## Environment List
 
-Each of the environments of your projects are available on the left sidebar.
+From your project’s main page, each of the environments are available from the pull-down menu ‘ENVIRONMENTS’ at the top of the page. 
+
+![Environment Pull-down](/images/env_pulldown.png)
+
+There is also a graphic view of your environments on the right hand side, where you can view your environments as a list or as a project tree. 
+
+![Environment Activity](/images/envs.png)
 
 The name of the environment is strikethroughed if it's been disabled. If it has an arrow next to it, this means the environment has children.
 
 ## Activity Feed
 
-The Web Interface displays all the activity happening on your environments. You can filter messages per type.
-
-## Header
+The management console displays all the activity happening on your environments. You can filter messages per type.
 
+![Environment Activity](/images/activity.png)
```

Comment:
We should take this opportunity to put more organization into the images directory, I think.  A "screenshots" directory?  Or a "management-console" directory?  Or something like that?  If we're changing all the links anyway, let's change them to something better organized and standardly-named.  (So consistently lower-case and dashes between words rather than underscores.)

Reply:
I have moved everything to a `mgmt-console` sub-dir and made a few others to clean the whole thing up. There is a separate `to-replace` sub-dir that sets aside some screenshots I know will be replaced soon (that I made), but that will soon be deleted to leave the following:

* images
   * cli
   * config-diagrams
   * integrations
   * mgmt-console
   * workflow

I can still move logo files around too if we want everything organized this way.